crypto/multisig/compact_bit_array_test.go

```diff
@@ -0,0 +1,169 @@
+package multisig
+
+import (
+	"encoding/json"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	cmn "github.com/tendermint/tendermint/libs/common"
+)
+
+func randCompactBitArray(bits int) (*CompactBitArray, []byte) {
+	numBytes := (bits + 7) / 8
+	src := cmn.RandBytes((bits + 7) / 8)
+	bA := NewCompactBitArray(bits)
```

Comment:
can't we just loop over bits, flip a coin, and Set if its true?

Reply:
Sure, the intended use of this was that it'd be faster, but I guess theres no speed concern. (RandBool internally calls Int63)